packages/destination-actions/src/destinations/adobe-target/updateProfile/__tests__/index.test.ts

```diff
@@ -0,0 +1,258 @@
+import nock from 'nock'
+import { createTestEvent, createTestIntegration } from '@segment/actions-core'
+import Destination from '../../index'
+
+const testDestination = createTestIntegration(Destination)
+const settings = {
+  client_code: 'segmentexchangepartn',
+  client_id: '123-test'
+}
+
+describe('AdobeTarget', () => {
+  describe('updateProfile', () => {
+    it('Handle a Basic Event', async () => {
+      nock(
+        `http://segmentexchangepartn.tt.omtrdc.net/rest/v1/profiles/thirdPartyId/${settings.client_id}?client=${settings.client_code}`
+      )
+        .get(/.*/)
+        .reply(200, {})
+      nock(
+        `https://${settings.client_code}.tt.omtrdc.net/m2/${settings.client_code}/profile/update?mbox3rdPartyId=${settings.client_id}`
+      )
+        .post(/.*/)
+        .reply(200, {})
+
+      const event = createTestEvent({
+        event: 'identify',
+        userId: '123-test',
+        properties: {
```

Comment:
```suggestion
        traits: {
```

`identify` events will have traits here rather than properties.